docs/app/components/documentation-category/documentation-category.component.html

```diff
@@ -1,2 +1,4 @@
 <uxd-component-section *ngFor="let section of category.sections" [id]="section.id" [title]="section.title" [componentName]="section.component"
-    [version]="section.version" [deprecated]="section.deprecated" [externalUrl]="section.externalUrl" [usage]="section.usage"></uxd-component-section>
+    [version]="section.version" [deprecated]="section.deprecated" [externalUrl]="section.externalUrl" [usage]="section.usage"
+    [hidden]="angular && section.version === 'AngularJS' && section.deprecated === true ||
```

Comment:
We should use `ngIf` instead of hidden as hidden still keeps everything in the dom and running so we aren't getting any of the performance benefits by hiding them. I realise there is an ngFor here so it will complain about using another structural directive on the same element, however you could move the ngIf to the template of the `uxd-component-section` perhaps